.github/workflows/testing.yml

```
@@ -139,4 +139,5 @@ jobs:
           file: ./codecov.xml
           flags: unittests
           name: codecov-umbrella
-          yml: ./codecov.yml
+          codecov_yml_path: ./codecov.yml
+          verbose: true
```

```suggestion
          fail_ci_if_error: true
          verbose: true
```

I'd also recommend this one

---

I wanted to do it but did not want to have that discussion right. Great!

---

Note that this will probably make the CI fail as it seems that tokens are not available on branches":

https://github.com/catalystneuro/neuroconv/actions/runs/7994541423/job/21832860179?pr=756

But I think we should solve this.

---

Right this is on-going:

https://github.com/catalystneuro/neuroconv/actions/runs/7994978795/job/21834447378?pr=759

---

It's not about it being a separate branch, it's about it not being passed from deployment

Though the real problem is that without that CI fail flag codecov never complains about such a major issue

---

What do you mean not being passed from deployment? 
It seems that is working on the CI actions that run from main.

---

> It seems that is working on the CI actions that run from main.

What makes you think that

---

I looked a the actions from main in the list and I can see that is being uploaded there:

https://github.com/catalystneuro/neuroconv/actions/runs/7976494254/job/21777198212

---

Mmmmm, yes

Because that was the daily cron trigger. Thankfully that never broke

Because it triggers from https://github.com/catalystneuro/neuroconv/blob/main/.github/workflows/testing.yml#L3 it doesn't need the CODECOV token passed

So I suppose in a roundabout way it is an issue affecting branches because they rely exclusively on the workflow call

But if I had, for instance, triggered a manual dispatch from `main` the same problem might have occured

---

Ah, got you, it is because it is called from from the `deploy-test.yml` and that does not propagate secrets.